scipy/interpolate/_cubic.py

```diff
@@ -164,15 +164,16 @@ class PchipInterpolator(CubicHermiteSpline):
 
     Parameters
     ----------
-    x : ndarray
+    x : ndarray, shape (npoints, )
         A 1-D array of monotonically increasing real values. ``x`` cannot
         include duplicate values (otherwise f is overspecified)
-    y : ndarray
-        A 1-D array of real values. ``y``'s length along the interpolation
-        axis must be equal to the length of ``x``. If N-D array, use ``axis``
-        parameter to select correct axis.
+    y : ndarray, shape (..., npoints, ...)
+        A N-D array of real values. ``y``'s length along the interpolation
+        axis must be equal to the length of ``x``. Use the ``axis``
+        parameter to select the interpolation axis.
     axis : int, optional
-        Axis in the y array corresponding to the x-coordinate values.
+        Axis in the ``y`` array corresponding to the x-coordinate values. Defaults
+        to ``axis=0``.
```

Comment:
So you wanted to stress that by default it's the first axis; do you want try keeping it two ways? _Defaults to the **first** axis, `axis=0`_ or some such?

Reply:
Instead, I stressed that unlike other interpolators, the interpolation axis in `interp1d` is the last one. So as far as I'm concerned I'm happy with this sentence